docs/meetings/newsletters/2024-01-04_volunteer_newsletter.md

```diff
@@ -0,0 +1,116 @@
+* **Event page improvements** [Don't show join/leave event buttons for past events](https://github.com/Couchers-org/couchers/issues/4174)
+* **Don't receive messages from empty accounts** [Add a setting for users to not receive messages from empty accounts](https://github.com/Couchers-org/couchers/issues/4089)
+* **Feedback slider defaults to negative** [Change it to default to positive to prevent accidenta negative ratings](https://github.com/Couchers-org/couchers/issues/3443)
+* **Profile image upload issues** [Clicking save while uploading terminate upload](https://github.com/Couchers-org/couchers/issues/5184)
+* **Add number of comments to events** [Add number of comments to event tiles](https://github.com/Couchers-org/couchers/issues/5217)
+* **Show user community events on dashboard** [Add events from city communities user belongs to on dashboard](https://github.com/Couchers-org/couchers/issues/5273)
+* **Upgrade bug reporting to include screenshot/video** [Upgrade bug reporting with sentry to include more context, photo, etc](https://github.com/Couchers-org/couchers/issues/5291)
+* **Frontend Package Upgrades**: Help us stay current by upgrading to [React v19](https://github.com/Couchers-org/couchers/issues/5381), [Next.js v14](https://github.com/Couchers-org/couchers/issues/5451), [Material-UI v6](https://github.com/Couchers-org/couchers/issues/5376), and [react-query v3](https://github.com/Couchers-org/couchers/issues/5263), among others.
+* 📱 **React Native Mobile App**: We could use more hands here. Help [Ivan](https://github.com/polemius) tackle the standalone app! We don't have a specific ticket for this as we're making it from scratch, see [how to get involved](#-how-to-get-involved) to help.
+
+Check out our [Couchers Engineering Task Board](https://github.com/Couchers-org/couchers/projects) for more and see [How To Get Involved](#-how-to-get-involved).
+
+---
+
+## 🎨 [Design & Operations Accomplishments](#-design-and-operations-accomplishments)
+
+* 📧 **Graphic Email Notifications**: We’ve moved from plain text emails to a polished, graphic design for notifications.
+* 📝 **Private Feedback Flow**: We’ve designed improvements for private feedback during the reference process (implementation pending). 
+
+Shoutout to [Jesse](https://couchers.org/user/jesse), who’s been doing design and support solo! Any Figma whizzes among you? Come help him!
+
+* ❓ **Couchers 2024 User Survey**: We conducted a survey among all Couchers.org users to gain insight into the main issues and priorities of our users. Improvements to the map search and filtering of inactive/incomplete profiles were the most requested feautures in the survey. Shoutout to [Yannic](https://github.com/spreeni) who had the main role in this!
+
+### ✍️ [Help Wanted: Design](#️-help-wanted-design)
```

Comment:
I'll have to find my little list of other things on my mind.

Need to add:

* Couchers Engineering logo
* Couchers stickers
* Mail verification postcard!

Reply:
Do you want these in the Help Wanted: Design section?

So we want to design an Couchers Engineering specific logo?